### ISSUE TITLE
Repacking can cause duplicate TypeForwards

### ISSUE THREAD

**Issue by gluck** [gluck/il-repack#143]
Not sure it was any (real) impact but it causes a PEVerify error:

```
.class extern /*2700001A*/ forwarder System.Collections.IStructuralComparable
{
  .assembly extern mscorlib /*23000004*/ 
}
.class extern /*2700001B*/ forwarder System.Collections.IStructuralEquatable
{
  .assembly extern mscorlib /*23000004*/ 
}
.class extern /*27000001*/ forwarder System.Collections.IStructuralEquatable
{
  .assembly extern mscorlib /*23000004*/ 
}
.class extern /*27000002*/ forwarder System.Collections.IStructuralComparable
{
  .assembly extern mscorlib /*23000004*/ 
}
```

```
[MD]: Error: ExportedType has a duplicate, token=0x2700001b. [token:0x27000001]
[MD]: Error: ExportedType has a duplicate, token=0x2700001a. [token:0x27000002]
[MD]: Error: ExportedType has a duplicate, token=0x27000002. [token:0x2700001A]
[MD]: Error: ExportedType has a duplicate, token=0x27000001. [token:0x2700001B]
```

**Comment by Alexx999** [comment]
From looks of it - this is what causes trouble in #203, should be fixed by #235

**Comment by KirillOsenkov** [comment]
We can reactivate if it's still an issue
